reference-implementation/to-upstream-wpts/writable-streams/bad-underlying-sinks.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+  self.importScripts('../resources/test-utils.js');
+  self.importScripts('../resources/recording-streams.js');
+}
+
+const error1 = new Error('error1');
+error1.name = 'error1';
+
+promise_test(t => {
+
```

Comment:
I am biased towards Chromium style. That may or may not correspond with what is appropriate for web-platform-tests.